docs/build/guides/transactions/create-account.mdx

```diff
@@ -11,278 +11,829 @@ import { Alert } from "@site/src/components/Alert";
 
 _Before we get started with working with Stellar in code, consider going through the following examples using the [Stellar Lab](https://lab.stellar.org). The lab allows you create accounts, fund accounts on the Stellar test network, build transactions, run any operation, and inspect responses from Horizon via the Endpoint Explorer._
 
-[Accounts](../../../learn/fundamentals/stellar-data-structures/accounts.mdx) are a fundamental building block of Stellar: they hold all your balances, allow you to send and receive payments, and let you place offers to buy and sell assets. Since pretty much everything on Stellar is in some way tied to an account, the first thing you generally need to do when you start developing is create one. This beginner-level tutorial will show you how to do that.
+[Accounts](../../../learn/fundamentals/stellar-data-structures/accounts.mdx) are a fundamental building block of Stellar: they hold all your balances, allow you to send and receive payments, and let you place offers to buy and sell assets. Since pretty much everything on Stellar is in some way tied to an account, the first thing you generally need to do when you start developing is create one. This beginner-level tutorial walks through the four building blocks you'll need: generating keys, funding an account, creating another account, and fetching balances.
```

Comment:
this sentence seemed like a run-on. here is one alternative for consideration.
```suggestion
[Account](../../../learn/fundamentals/stellar-data-structures/accounts.mdx) is a fundamental building block of Stellar. It provides secure identity and maintains balances of your assets and is required to participate on the Stellar network. This beginner-level tutorial walks through several key concepts of an account: generating key pair, funding, creating another account, and fetching balances.
```

Reply:
After reviewing, wanted to revisit this section with a follow-on suggestion - break each of these four concepts into sub-bullets with their copy and then consolidate to a single example program afterwards which performs all:

```
Create an Account:
  * Create a keypair - copy here...
  * Funding  - copy here...
  * Create another Account - copy here...
  * Fetch Balances - copy here ...
  
Examples:
  js, go, java, python
     
Now that you’ve got an account ....
```

Reply:
You mean don't have examples per-section and just have copy there, then one big example at the end?

Reply:
Honestly a great idea. Gave it a shot in 0d5024b950bc8c185cfe10c4e9d54da53fb184ff, wdyt?

Reply:
lgtm. concise!

only nit is the mention of 'four' building blocks but it's down to 3 now. 
do you want to have consistent link texts matching the section titles?  i.e. the building block link is stated as 'generating keys' and the linked section title is  slightly different 'Create a Keypair'

Reply:
I’m okay with the “mid sentence copy” vs “title copy” being different! Fitting the titles exactly ruins the flow.